runtime-modules/content/src/lib.rs

```diff
@@ -674,10 +674,13 @@ decl_module! {
 
             // upload to storage
             if let Some(upload_assets) = params.assets.as_ref() {
+
+        // upload assets to storage after creating eth bag
                 Self::upload_assets_to_storage(
```

Comment:
No checks for dynamic bag creation and object uploading before the `mutation safe` mark.

Reply:
Ok